### ISSUE TITLE
`check-alignment` has false-positive when not using star prefixes

### ISSUE THREAD

**Issue by sindresorhus** [gajus/eslint-plugin-jsdoc#1351]
## Expected behavior

I expected this to be allowed:

```js
/**
@param {string} text
*/
const a = x => x;
```

When not using star prefix, the proper alignment means that the start and end of the comment don't have any indentation.

For example, this should be caught:

```js
/**
@param {string} text
 */
const a = x => x;
```

## Actual behavior

It complains.

## ESLint Config

```js
{
  'jsdoc/check-alignment': 'error'
}
```

## ESLint sample

Already shared.

## Environment

- Node version: 22.12.0
- ESLint version 9.18
- `eslint-plugin-jsdoc` version: 50.6.2

**Comment by brettz9** [comment]
That would be contrary to how the rule currently works. The rule tries to align against the first asterisk in a straight line. It is also the pattern used on the [JSDoc site](https://jsdoc.app/about-getting-started).

I suppose we could in theory add an option to enforce the following pattern which dropped the space (or potentially others like aligning on the second asterisk):

```js
/**
* // With asterisks
*/
function quux () {}
```

```js
/**
 // Without asterisks
*/
function quux () {}
```

I'm personally not inclined to work on it, and I'm personally a little ambivalent about encouraging deviation from more standard patterns, but given our general support for different styles, I suppose we could accept a PR.
